pom.xml

```diff
@@ -8,6 +8,31 @@
     <groupId>com.wavefront</groupId>
     <artifactId>java-sdk</artifactId>
     <version>1.0-SNAPSHOT</version>
```

Comment:
We decided to go with 0.9.0. Can you please change the version to 0.9.0-SNAPSHOT.